src/color/color.typ

```
@@ -1,32 +1,138 @@
-#let env-colors-list  = ("bootstrap", "bw", "gruvbox-dark", "ayu-light")
-#let env-colors       = state("theme", "bootstrap")
-#let colors-dict      = (:)
+#let ergo-colors-list = ("bootstrap", "bw", "gruvbox-dark", "ayu-light")
+#let ergo-colors      = (:)
 
-#for colors-name in env-colors-list {
-  colors-dict.insert(colors-name, json(colors-name + ".json"))
+// Initialize colors
+#for colors-name in ergo-colors-list {
+  ergo-colors.insert(colors-name, json(colors-name + ".json"))
 }
 
+// Checks if dict is valid colorscheme
```

Instead of returning a bool, can we panic on whatever failed, or maybe return a string indicating where the failure was and deal with it later? Because it's not clear how to bugfix a broken color scheme like this currently

---

Revisit in `0.3.0`